README_RISE.md

```diff
@@ -0,0 +1,55 @@
+#####Instrukcja instalacji wtyczki RISE służącej do obsługi interaktywnego pokazu slajdów
+
+
+# UNIX 
+
+1. Pobierz potrzebne pliki z repozytorium https://github.com/damianavila/RISE.git
+```
+cd
+git clone https://github.com/damianavila/RISE.git
```

Comment:
A jak sie zawartosc repozytorium RISE pozmienia to instrukcja dalej bedzie dzialac ?
Prosze zwrocic uwage na to: https://github.com/damianavila/RISE/releases